src/extendable_pydantic/_patch.py

```diff
@@ -66,12 +66,13 @@ def _create_response_field_wrapper(wrapped, instance, args, kwargs):
         _resolve_model_fields_annotation([field])
         return field
 
-    # fastapi <  0.112.3
-    wrapt.wrap_function_wrapper(
-        utils, "create_response_field", _create_response_field_wrapper
-    )
-
-    # fastapi >= 0.112.3
-    wrapt.wrap_function_wrapper(
-        utils, "create_model_field", _create_response_field_wrapper
-    )
+    if hasattr(utils, "create_response_field"):
+        # For fastapi < 0.112.3
+        wrapt.wrap_function_wrapper(
+            utils, "create_response_field", _create_response_field_wrapper
+        )
+    else:
```

Comment:
```suggestion
    if hasattr(utils, "create_model_field"):
```

would be more robust?

Reply:
Thank @sbidoul